remote_execution/oss/re_grpc/src/client.rs

```diff
@@ -764,17 +1081,26 @@ where
             }
         }
 
-        async {
+        let fut = async {
             let mut file = opts
                 .open(&req.named_digest.name)
                 .await
                 .context("Error opening")?;
-            file.write_all(&data).await.context("Error writing")?;
+            // If the data is small enough to be transfered in a batch
+            // blob update, just write it directly to file.
+            if req.named_digest.digest.size_in_bytes < max_msg_size as i64 {
+                let data = get(&req.named_digest.digest)?;
+                file.write_all(&data).await.context("Error writing")?;
+            } else {
+                let mut responses = big_f(req.named_digest.digest.clone()).await?;
+                while let Some(ReadResponse { data }) = responses.message().await? {
+                    file.write_all(&data).await.context("Error writing")?;
+                }
```

Comment:
Can this get some `context` on most calls so that if this errors out we know what happened?